src/program/snabbvmx/doc/README.md

```diff
@@ -0,0 +1,112 @@
+# SnabbVMX
+
+## What's SnabbVMX?
+
+SnabbVMX is a network design that combines Snabb's lwAFTR app and a VM managed
+by Snabb.  For a detailed explanation of what's an lwAFTR, please refer to
+lwAFTR docs.
+
+Snabb's lwAFTR app, in addition to managing lw4o6 traffic (IPv4 and IPv6 packets),
+it also manages non-lwAFTR traffic such as pings to lwAFTR interfaces, ARP and
+NDP resolution.  This traffic is managed by addition applications connected to
+the lwAFTR network design in program/lwaftr/run.lua.  Management of this type
+of traffic is a requiremennt in RFC 7596.
+
+SnabbVMX uses a different approach.  It setups a network design where only lw4o6
+packets are managed by the lwAFTR app, while everything else is forwarded to a VM
+which is in charge of providing ping, ARP and NDP resolution.
+
+## App network
+
+SnabbVMX's app network:
+
+![SnabbVMX](.images/snabbvmx.png)
```

Comment:
It seems I needed to `git -f add` the image to actually add it. Now it's showing a link (the image is actually a symbolic link to an image in the parent folder).